valhalla/mjolnir/landmark_database_builder.h

```diff
@@ -66,6 +65,14 @@ struct LandmarkDatabase {
   void release_prepared_stmt();
 };
 
+// class?
+struct LandmarkParser {
+public:
+  static std::vector<Landmark> Parse(const std::vector<std::string>& input_files);
+};
```

Comment:
you can move this to the implementation file, its not needed for the public api, only the build function is needed which will make use of this parser callback thing, similar to the admin processing

Reply:
oh i see this is the class analog. so in this case because you dont need any state, you can totally just remove this and put the guts of the work into the build function! sorry for the misunderstanding

Reply:
done.